Sources/Testing/Testing.docc/known-issues.md

```diff
@@ -126,9 +136,10 @@ known:
 }
 ```
 
-If the underlying problem is truly nondeterministic, you may acknowledge this
-and instruct `withKnownIssue()` to not record an issue indicating there were no
-known issues by passing `isIntermittent: true`:
+If the underlying problem is unpredictable and fails at random, you can pass
+`isIntermittent: true` to let the testing library know that it will not always
+occur. Then, the testing library will _not_ record the issue indicating that no
```

Comment:
Double negatives are hard to read. How about:

> By default, if a known issue is expected to occur and does not, the testing library records an issue to let you know that the test's behavior has changed. If the underlying problem is unpredictable or fails at random, you can pass `isIntermittent: true` to let the testing library know that it will not always occur and that it should not record this additional issue.

Reply:
Pushed an attempt to phrase it more elegantly now